### ISSUE TITLE
aarch64 support

### ISSUE THREAD

**Issue by PythonNut** [jcs-elpa/flx-rs#9]
Hi! Support for Apple Silicon was brought up in #8. The solution suggested there was to use universal binaries on macOS. For Linux and Windows, I am not sure what the best solution would be.

**Comment by jcs090218** [comment]
We are building binaries through GitHub Actions. Some references I have found,

* https://patriksvensson.se/posts/2020/05/targeting-arm-for-windows-in-rust
* https://github.com/emacs-tree-sitter/elisp-tree-sitter/blob/d149a3f1340898792ed60af6b1bbe82d03090c82/core/tsc-dyn-get.el#L83-L92

It shouldn't be too hard to implement.

**Comment by jcs090218** [comment]
There is still an issue with `aarch64-unknown-linux-gnu` build, but otherwise should be good with #11.


<details>
    <summary>Error log for `aarch64-unknown-linux-gnu`</summary>

```
warning: unused `Result` that must be used
Warning:   --> src/dynmod.rs:41:9
   |
41 |         vec.set(index, data);
   |         ^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_must_use)]` on by default
   = note: this `Result` may be an `Err` variant, which should be handled
error: linking with `cc` failed: exit status: 1
Error:   |
  = note: "cc" "-Wl,--version-script=/tmp/rustcjDm7tR/list" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.36lmnffn4zypvb5s.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.addr2line-484db03cffcca5b6.addr2line.82068c91-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.adler-91ebd391b9f3e29e.adler.1b0aee10-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.alloc-cc0694ace9be0e66.alloc.af863b8c-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.10.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.11.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.12.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.13.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.14.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.15.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.2.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.3.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.4.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.5.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.6.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.7.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.8.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.anyhow-67999c4dfdc9c9eb.anyhow.fd5b298d-cgu.9.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.cfg_if-12a6b9815d90b1e4.cfg_if.79c4b9bd-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.core-d1ce211496f525c3.core.9ff7c999-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.10.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.11.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.12.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.13.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.14.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.15.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.2.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.3.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.4.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.5.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.6.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.7.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.8.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs-c98f9cd4f3e4cacf.emacs.4d05cc3e-cgu.9.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs_module-4a799a06afd6a209.emacs_module.c8defda1-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs_module-4a799a06afd6a209.emacs_module.c8defda1-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs_module-4a799a06afd6a209.emacs_module.c8defda1-cgu.2.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs_module-4a799a06afd6a209.emacs_module.c8defda1-cgu.3.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs_module-4a799a06afd6a209.emacs_module.c8defda1-cgu.4.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.emacs_module-4a799a06afd6a209.emacs_module.c8defda1-cgu.5.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.10.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.11.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.12.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.13.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.14.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.15.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.2.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.3.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.4.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.5.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.6.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.7.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.8.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs-703f7d1fc1703869.flx_rs.9f48085f-cgu.9.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.0.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.1.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.10.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.11.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.12.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.13.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.14.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.15.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.2.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.3.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.4.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.5.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.6.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.7.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.8.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.flx_rs_core.04d4ef29-cgu.9.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.gimli-77337fc21bd1cca4.gimli.0a5593d5-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.hashbrown-f5e9f9d7692ae465.hashbrown.480ad548-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.libc-b0a59add1f3108e8.libc.6df625ce-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.memchr-25c2a8eab25c758c.memchr.cde62fd4-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.miniz_oxide-0ae529ce319bd2e0.miniz_oxide.1af5748d-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.object-59ae13aa5b85e878.object.1b79d6a8-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.2.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.3.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.4.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.5.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.6.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.7.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.8.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.once_cell-7b0b14704297d9ac.once_cell.63755920-cgu.9.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.panic_unwind-2de0d40477[269](https://github.com/jcs-elpa/flx-rs/runs/7117106832?check_suite_focus=true#step:5:270)ad1.panic_unwind.dcc6c003-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.rustc_demangle-8399fd816afb0bb6.rustc_demangle.bf2b4237-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.rustc_std_workspace_alloc-79cdd974e440af93.rustc_std_workspace_alloc.80dd326b-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.rustc_std_workspace_core-5a84de01a2ad7b8c.rustc_std_workspace_core.91eab6f0-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.std-7d5a620ab77655ab.std.83ea3440-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.std_detect-dbd9d453a2c2347c.std_detect.27a07f73-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.thiserror-15dc2acd6dbb86fc.thiserror.49d38cc2-cgu.0.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.thiserror-15dc2acd6dbb86fc.thiserror.49d38cc2-cgu.1.rcgu.o.rcgu.o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.unwind-fb4f5845224d8c3b.unwind.06aba5aa-cgu.0.rcgu.o.rcgu.o" "-Wl,--as-needed" "-L" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps" "-L" "/home/runner/work/flx-rs/flx-rs/core/target/release/deps" "-L" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-gnu/lib" "-Wl,--start-group" "-Wl,--end-group" "-Wl,-Bstatic" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-gnu/lib/libcompiler_builtins-9453083e1ae0beda.rlib" "-Wl,-Bdynamic" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-L" "/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-gnu/lib" "-o" "/home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/libflx_rs_core.so" "-Wl,--gc-sections" "-shared" "-Wl,-zrelro,-znow" "-Wl,-O1" "-nodefaultlibs"
  = note: /usr/bin/ld: /home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.36lmnffn4zypvb5s.rcgu.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.36lmnffn4zypvb5s.rcgu.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.36lmnffn4zypvb5s.rcgu.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.36lmnffn4zypvb5s.rcgu.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.36lmnffn4zypvb5s.rcgu.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.36lmnffn4zypvb5s.rcgu.o: Relocations in generic ELF (EM: 183)
          /usr/bin/ld: /home/runner/work/flx-rs/flx-rs/core/target/aarch64-unknown-linux-gnu/release/deps/flx_rs_core.36lmnffn4zypvb5s.rcgu.o: error adding symbols: file in wrong format
          collect2: error: ld returned 1 exit status
warning: `flx_rs_core` (lib) generated 1 warning
Warning: error: could not compile `flx_rs_core` due to previous error; 1 warning emitted
Error: Process completed with exit code 101.
```

</details>

**Comment by chriselrod** [comment]
I did not get the error when building locally with `cargo build --release` on aarch64 linux (running Asahi linux).

**Comment by jcs090218** [comment]
I think the issue was cross-compile between platforms. Unfortunately, there is no way to specify CPU architecture in GHA. Would you like to make a PR to submit a binary to our [bin](https://github.com/jcs-elpa/flx-rs/tree/master/bin) folder?

**Comment by jcs090218** [comment]
Fixed in #19.
